api/bertymessenger.proto

```diff
@@ -223,11 +223,13 @@ message AppMessage {
   message GroupInvitation {
     string link = 2; // TODO: optimize message size
   }
-  message SetGroupName {
-    string name = 2;
+  message SetGroupInfo {
+    string display_name = 1;
+    string avatar_cid = 2; // TODO: optimize message size
```

Comment:
I think we should add a `bool update_all` field or split the GroupInfo call for each available field:
* SetGroupName
* SetGroupAvatarCID

This is one of the rare "shared" entity, and because of the synchronicity of Berty in general, it's safer to allow someone to change just the title without requiring to be up-to-date for the other fields

To keep the API small, we can just keep `SetGroupInfo` and only set what is `!= ""`, in that case, we only need to make sure that the front code will only call SetGroupInfo with the "just-changed field" and not with all the group state known by the app

Reply:
>  only set what is != ""
this is what I'm doing (or tried at least ^^)